XMonad/Actions/WindowGo.hs

```diff
@@ -66,12 +67,20 @@ appropriate one, or cover your bases by using instead something like:
 For detailed instructions on editing your key bindings, see
 "XMonad.Doc.Extending#Editing_key_bindings". -}
 
+workspacesSorted :: Ord i => W.StackSet i l a s sd -> [W.Workspace i l a]
+workspacesSorted s = L.sortBy (\u t -> W.tag u `compare` W.tag t) $
+                       W.workspaces s
```

Comment:
Thanks for your feedback. I fixed the indentation, and added a comment to explain the purpose of workspacesSorted.